packages/flutter_tools/lib/src/windows/topaz_ofd_validator.dart

```diff
@@ -0,0 +1,41 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../base/io.dart';
+import '../doctor_validator.dart';
+
+const String kCoreProcessPattern = r'Topaz OFD\\Warsaw\\core.exe';
+
+class TopazOfdValidator extends DoctorValidator {
+  const TopazOfdValidator({required this.processLister}) : super('Topaz OFD');
+
+  final ProcessLister processLister;
+
+  @override
+  Future<ValidationResult> validate() async {
+    final String tasks = await processLister.getProcessesWithPath('core');
+    final RegExp pattern = RegExp(kCoreProcessPattern, multiLine: true, caseSensitive: false);
+    final bool matches = pattern.hasMatch(tasks);
+    if (matches) {
+      return const ValidationResult(
+        ValidationType.partial,
+        <ValidationMessage>[
+          ValidationMessage.hint('The Topaz OFD Security Module process has been found running. If you are unable to build, you will need to disable it.'),
+        ],
+        statusInfo: 'Topaz OFD may be running');
+    } else {
+      return const ValidationResult(
+        ValidationType.success,
+        <ValidationMessage>[]);
+    }
+  }
+}
+
+class ProcessLister {
```

Comment:
Looks like this can have a const constructor

Reply:
Not anymore now that we inject the process manager.

Reply:
fair :)

Reply:
Can you remove the changes request if this is resolved?

Reply:
As I recall, the only way in GitHub to undo a "requested changes" state is to approve. I re-requested review, then removed myself as a reviewer, and it still shows I requested changes. I will approve once Elias does.